clifford/test/test_numba_extensions.py

```diff
@@ -174,15 +174,15 @@ def test_pickling():
     assert pickle.loads(pickle.dumps(lt)) is lt
 
 
-def test_F_order():
+def test_A_order():
     import numpy as np
 
     @numba.njit
-    def mul_mv(a):
-        return a*e3
+    def mul_mv(mv):
+        return mv*e3
 
-    mva = layout.MultiVector(np.ones(8, order='F'))
-    mvb = layout.MultiVector(np.ones(8, order='C'))
+    mva = layout.MultiVector(np.ones(8)[::-1])
```

Comment:
```suggestion
    mva = layout.MultiVector(np.ones(8)[::-1])
    assert not mva.value.c_contiguous
```
to check this is actually testing what you want to test

Reply:
Also, `8` should be `layout.gaDims`, because you're in g3c here.